### PR TITLE
Add support for 'tz' parameter of Graphite Render API

### PR DESCRIPTION
**Pull request by tkirill**
Docs: https://graphite.readthedocs.io/en/latest/render_api.html#tz

ahdde/graphite.net#36

### CODE REVIEW

File: Graphite/GraphiteClient.cs [hunk 1]
```diff
@@ -162,8 +163,9 @@ public Task<GraphiteMetricData[]> GetMetricsDataAsync(SeriesListBase target, str
         /// <param name="template">The target metrics can use a special <see cref="SeriesListBase.Template(string[])"/> function which allows the metric paths to contain variables</param>
         /// <param name="maxDataPoints"></param>
         /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None"/>.</param>
+        /// <param name="tz">specify time zone for from/until. Corresponds to 'tz' parameter in Graphite Render API.</param>
         /// <returns></returns>
-        public async Task<GraphiteMetricData[]> GetMetricsDataAsync(SeriesListBase[] targets, string from = null, string until = null, IDictionary<string,string> template = null, ulong? maxDataPoints = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<GraphiteMetricData[]> GetMetricsDataAsync(SeriesListBase[] targets, string from = null, string until = null, IDictionary<string,string> template = null, ulong? maxDataPoints = null, CancellationToken cancellationToken = default(CancellationToken), string tz = null)
```
Comment (zivillian) [inline Graphite/GraphiteClient.cs:168]: can we make the cancellationToken the last parameter?

Comment (tkirill) [reply Graphite/GraphiteClient.cs:168]: Yes but it will break existing usages. If you're OK with it I'll do it.

Comment (tkirill) [reply Graphite/GraphiteClient.cs]: fixed :)

File: Graphite/GraphiteClient.cs [hunk 2]
```diff
@@ -146,11 +146,12 @@ public async Task<string[]> ExpandMetricsAsync(bool leavesOnly, CancellationToke
         /// <param name="until">specify the relative or absolute end to graph</param>
         /// <param name="template">The target metrics can use a special <see cref="SeriesListBase.Template(string[])"/> function which allows the metric paths to contain variables</param>
         /// <param name="maxDataPoints"></param>
+        /// <param name="tz">specify time zone for from/until. Corresponds to 'tz' parameter in Graphite Render API.</param>
```
Comment (zivillian) [inline Graphite/GraphiteClient.cs:149]: Can we use the description from the graphite docs?
> Time zone to convert all times into.

Comment (tkirill) [reply Graphite/GraphiteClient.cs]: Good idea! Fixed.

**Pull request merged**